### PR TITLE
feat: Modify Withdrawal Transaction for Unbonded Validators

### LINKED ISSUE #464: Modify Withdrawal Transaction for Unbonded Validators
## Description

The fee-less withdrawal transaction for unbonded validators may encourage spamming attacks on the network. To address this issue, modify the transaction to only allow the withdrawal of the entire stake amount from the validator's address to the designated receiver account. This would discourage spamming attacks without imposing additional fees for withdrawals, which may be beneficial for validators who have already paid fees for bonding.

### Expected Behavior

Withdrawal transaction only withdraws the entire stake amount from the validator's address to the designated receiver account, discouraging spamming attacks on the network.

### PR DESCRIPTION
**Pull request by kehiy**
## Description
I added a check to make sure every withdrawal transaction has an amount equal to the stake amount to prevent spam attacks.
 
## Related issue(s)
- Fixes #464

### CODE REVIEW

File: execution/executor/withdraw.go [hunk 1]
```diff
@@ -29,8 +29,9 @@ func (e *WithdrawExecutor) Execute(trx *tx.Tx, sb sandbox.Sandbox) error {
 		return errors.Errorf(errors.ErrInvalidSequence,
 			"expected: %v, got: %v", val.Sequence()+1, trx.Sequence())
 	}
-	if val.Stake() < pld.Amount+trx.Fee() {
-		return errors.Error(errors.ErrInsufficientFunds)
+	if val.Stake() != pld.Amount+trx.Fee() {
```
Comment (amirvalhalla) [inline execution/executor/withdraw.go:32]: why it should be `!=` and not `<` ?

**Review by amirvalhalla — CHANGES_REQUESTED** [review]
@kehiy 
please resolve request changes

File: execution/executor/withdraw.go (hunk 1 above)
Comment (kehiy) [reply execution/executor/withdraw.go:32]: check the issue

Comment (amirvalhalla) [reply execution/executor/withdraw.go:32]: @kehiy 
thanks, please fix the pipeline errors.

**Review by amirvalhalla — APPROVED** [review]

**Review by amirvalhalla — APPROVED** [review]

**Review by amirvalhalla — APPROVED** [review]

**Comment by amirvalhalla** [comment]
@kehiy 
Hi, what is the final status of the PR? as long as time goes, the conflicts will increase, it would be better to fix issues and merge PR.
thanks

**Comment by kehiy** [comment]
Hi @amirvalhalla,
We need more research and discussion with @b00f about this.

About conflicts and other stuff don't worry.

We will merge/close it after discussion and researches.

Thanks for reminding us (:

**Pull request closed without merging**